utils_cv/detection/data.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import List
+from urllib.parse import urljoin
+
+
+class Urls:
+    # for now hardcoding base url into Urls class
+    base = (
+        "https://cvbp.blob.core.windows.net/public/datasets/object_detection/"
+    )
+
+    # traditional datasets
+    fridge_objects_path = urljoin(base, "odFridgeObjects.zip")
+
+
+def get_coco_labels() -> List[str]:
```

Comment:
Maybe just call coco_labels(), similar to the imagenet_labels() function for classification